phpunit.xml.dist

```diff
@@ -44,6 +45,7 @@
                 <directory>./src/Symfony/Bridge/*/Tests</directory>
                 <directory>./src/Symfony/Component/*/Tests</directory>
                 <directory>./src/Symfony/Component/*/*/Tests</directory>
+                <directory>./src/Symfony/Contract/*/Tests</directory>
```

Comment:
Is there a specific reason why these rules aren't sorted?

Reply:
same a s above